packages/marko-web-gam/browser/inject-ads.vue

```diff
@@ -67,7 +67,9 @@ export default {
               $child.nextAll(childSelector).each(function handleBefore() {
                 if ($(this).text().length > 1) {
                   const $previous = $(this).prev();
-                  if (component.detectEmbeds && $previous.attr('data-embed-type')) {
+                  if ($previous.is('h1,h2,h3,h4,h5,h6')) {
```

Comment:
Make this header list a variable and re-use throughout

Reply:
@zarathustra323 - done.